tests/test_utils/test_timer.py

```diff
@@ -18,10 +18,11 @@ def test_timer_init():
 def test_timer_run():
     timer = mmengine.Timer()
     time.sleep(1)
-    assert abs(timer.since_start() - 1) < 1e-2
+    # In windows, the error could be larger than 20ms.
```

Comment:
```suggestion
    # In Windows, the error could be larger than 20ms.
```

Reply:
Details (https://stackoverflow.com/questions/11657734/sleep-for-exact-time-in-python) about the sleep time in Windows.